src/Grpc.Net.Client/Internal/GrpcDiagnostics.cs

```diff
@@ -0,0 +1,33 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System.Diagnostics;
+
+namespace Grpc.Net.Client.Internal
+{
+    internal static class GrpcDiagnostics
+    {
+        // This is a static on a non-generic class so it isn't re-created once for each type argument
+        public static readonly DiagnosticListener DiagnosticListener = new DiagnosticListener("GrpcCallDiagnosticListener");
+
+        public const string ActivityName = "Grpc.Net.Client.GrpcOut";
```

Comment:
I followed what I saw here - https://github.com/aspnet/AspNetCore/blob/6ecfdf2c49ffe25ebc3e8872d186bb85fe8e26b9/src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs#L18

So you think the name should just be `GrpcOut`?

Reply:
Someone could look at the activity without using the DiagnosticSource. Wouldn't the namespace still be useful in that situation?

Reply:
yes, I meant `GrpcOut`.

you are right, someone can try creating/using activities without DiagnosticSource (e.g. logger) and name might be useful there in some cases. 

At the same time, Activity name so far is only used as a part of DiagnosticSource event (to subscribe, filter and apply custom payload parsing in ApplicationInsights or OpenTelemetry), so I doubt it will be useful without DS.

I'm ok keeping it to `Grpc.Net.Client.GrpcOut` if you think it may be useful.